src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -3206,6 +3208,53 @@ def test_api_video_update_live_state_stopped(self):
             },
         )
 
+    @override_settings(UPDATE_STATE_SHARED_SECRETS=["shared secret"])
+    @override_settings(LIVE_CHAT_ENABLED=True)
+    def test_api_video_update_live_state_stopped_missing_manifest(self):
+        """Updating state to stopped should set video DELETED if manifest is missing."""
```

Comment:
"""Updating state to stopped should set video **to** DELETED if the manifest is missing."""